addon-test-support/setup-global-a11y-hooks.ts

```diff
@@ -1,4 +1,4 @@
-import { _registerHook } from '@ember/test-helpers';
+import { registerHook } from '@ember/test-helpers';
```

Comment:
We should also import `HookUnregister` directly instead of defining it inline further below.
```suggestion
import { registerHook, type HookUnregister } from '@ember/test-helpers';
```

Reply:
resolved!